source/playgrounds.txt

```diff
@@ -121,6 +122,25 @@ panel, the |vsce| *Overview* page, or the |vscode| Command Pallette.
 
       .. include:: /includes/steps/open-new-playground.rst
 
+   .. tab:: Your Project File
+      :tabid: project-file
+
+      1. Open a project file.
+
+         To open a file in |vscode-short|, click the file name in the
+         file explorer. 
+
+      #. Highlight the code that you want to open in a playground.
+
+      #. Right-click to select :guilabel:`Export Code to Playground`.
+
+         Right-click your selected code to open the options menu.
+         Click :guilabel:`MongoDB Copilot Participant` >
```

Comment:
In the immediate update coming up we're renaming this to "MongoDB Copilot Extension" instead of "Participant". Let's change it here so we don't have to amend shortly after.